auditlog/models/rule.py

```diff
@@ -485,6 +485,10 @@ def write_full(self, vals, **kwargs):
             if self.env.user in users_to_exclude:
                 return result
 
+            self.env[records_write._name].flush_model(
```

Comment:
Maybe

```suggestion
           self.flush_recordset(
```

is even more precise?

Reply:
Good point. Changed.